pkg/config/remote_config_test.go

```diff
@@ -0,0 +1,120 @@
+package config
```

Comment:
please use github.com/stretchr/testify/require here for assertions instead of directly invoking t.Error/t.FailNow :)